applications/DEM_application/test_examples/DEM_run_all_benchmarks.py

```diff
@@ -75,43 +75,43 @@ def Run():
         try:
             if platform.system()=="Windows":
                 os.system("setenv OMP_NUM_THREADS 1") # Is that the correct way to run on Windows?
-                subprocess.check_call(["python", path + "/DEM_benchmarks.py", str(benchmark), ">", "BenchTemp.txt"], stdout=f, stderr=f)
+                subprocess.check_call(["python", path + "/DEM_benchmarks.py", str(benchmark), ">", "BenchTemp.info"], stdout=f, stderr=f)
                 os.system("setenv OMP_NUM_THREADS 16") # Trying to set a 'default' value
```

Comment:
This default value is wrong. I propose resetting the variable (then the code uses the max number of threads). Let's take the chance to fix this now.

Reply:
I guess OMP_DYNAMIC=true will do the job